src/lib/y2packager/widgets/product_license_translations.rb

```diff
@@ -68,15 +71,61 @@ def handle(event)
       # @return [Y2Packager::Widgets::SimpleLanguageSelection]
       def language_selection
         @language_selection ||=
-          Y2Packager::Widgets::SimpleLanguageSelection.new(product.license_locales, language)
+          Y2Packager::Widgets::SimpleLanguageSelection.new(available_locales, content_language)
       end
 
-      # Product  selection widget
+      # Product selection widget
       #
       # @return [Widgets::ProductLicenseContent]
       def product_license
         @product_license ||=
-          Y2Packager::Widgets::ProductLicenseContent.new(product, language)
+          Y2Packager::Widgets::ProductLicenseContent.new(product, content_language)
+      end
+
+      # Available license translations
+      #
+      # When running on textmode, only the preselected/given language is considered.
+      # see #default_language for further details.
+      #
+      # @return [Array<String>] Locale codes of the available translations
+      # @see #default_language
+      def available_locales
+        return product.license_locales unless Yast::UI.TextMode
```

Comment:
Maybe for the consistency with the method below use the triple operator here as well:
```ruby
Yast::UI.TextMode ? [default_language] : product.license_locales
```

this also avoid `unless` which is often confusing...